pkg/types/ccip/models.go

```diff
@@ -18,3 +63,40 @@ type TxMeta struct {
 	TxHash                  string
 	LogIndex                uint64
 }
+
+// EIP55 converts the provided address to EIP55.
+//
+// If the ith digit is a letter (ie. it’s one of abcdef) print it in uppercase if the 4*ith bit of the hash of the
+// lowercase hexadecimal address is 1 otherwise print it in lowercase.
+func EIP55(addr string) (string, error) {
+	addr = strings.ToLower(strings.TrimPrefix(addr, "0x"))
```

Comment:
Since `0x` hex strings as addresses are pretty much exclusively Ethereum addresses, I wonder if we can gate this function on that as well, i.e `if !strings.StartsWith(addr, "0x") { return error }`

Reply:
fixed